tests/runtest.py

```diff
@@ -110,19 +107,22 @@
 global runDriver
 
 try:
-    driver_v = "NULL"
+    driver_v
 except NameError:
     from driverG import webDriver
     print("Running Selenium Grid")
     runDriver = webDriver(grid_server_ip)
-
 else:
-    if (driver_v == "U"):
+    if driver_v == "F":
         from driverU import webDriver
-        print ("Running Ubuntu driver")
+        print ("Running webDriver driver")
```

Comment:
driverU is for Ubuntu driver

Reply:
Actually, I meant Firefox I forgot to change it.

Reply:
Actually, driverU should support all system. I did some change.